src/spikeinterface/widgets/template_similarity.py

```diff
@@ -19,12 +19,18 @@ class TemplateSimilarityWidget(BaseWidget):
     display_diagonal_values : bool
         If False, the diagonal is displayed as zeros.
         If True, the similarity values (all 1s) are displayed. Default False
-    cmap : Matplotlib colormap
-        The matplotlib colormap. Default 'viridis'. (matplotlib backend)
+    cmap : str
```

Comment:
I don't know how the plot_data dict works but can you pass an actual cmap? Does it only accept string?

Reply:
Hi guys, I think it's only string!